### PR TITLE
link: Make identical bh-template as bemhtml

### LINKED ISSUE #443: link: Make identical bh-template as bemhtml

### PR DESCRIPTION
**Pull request by sipayRT**
Close #443 

@mishanga

### CODE REVIEW

File: common.blocks/link/link.bh.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = function(bh) {
 
         typeOfUrl !== 'undefined' && (attrs.href = typeOfUrl === 'string'?
             url :
-            url); // TODO: реализовать возможность отдавать bemjson в url
+            bh.apply(url));
```
Comment (mishanga) [inline common.blocks/link/link.bh.js:17]: `ctx.apply(url);`

**Comment by mishanga** [comment]
В остальном ок.

File: common.blocks/link/link.bh.js (hunk 1 above)
Comment (sipayRT) [inline common.blocks/link/link.bh.js:17]: at this case `ctx.apply()` returned `[object Object]` instead of `/action`.

Comment (mishanga) [inline common.blocks/link/link.bh.js:17]: Oops! Yes, of course, you are right.

**Pull request merged**